### DISCUSSION TITLE
v2.2

### DISCUSSION THREAD

**Discussion by port19x** [pystardust/ani-cli#713] [General] [answered]
## What's Changed
- Documentation
  - advise manual installation for Mac OS by @iamchokerman in https://github.com/pystardust/ani-cli/pull/671
  - advise install via curl instead of git by @iamchokerman in https://github.com/pystardust/ani-cli/pull/681
  - pull request template by @iamchokerman and @port19x in https://github.com/pystardust/ani-cli/pull/695
  - add packaging status section to readme by @WillPower3309 in https://github.com/pystardust/ani-cli/pull/707
  - added matrix and revolt to README by @iamchokerman in https://github.com/pystardust/ani-cli/pull/700
  - advise usage of native termux package by @port19x in https://github.com/pystardust/ani-cli/pull/700
 
- Bugfixes
  - new decryption algorithm by @CoolnsX in https://github.com/pystardust/ani-cli/pull/664
  - revert gogo encryption algo by @TheGlockMisc in https://github.com/pystardust/ani-cli/pull/670
  - update keys parsing by @CoolnsX in https://github.com/pystardust/ani-cli/pull/673
  - updated link parsing by @CoolnsX in https://github.com/pystardust/ani-cli/pull/677
  - fixed only audio playing for some anime by @RaynardGerraldo in https://github.com/pystardust/ani-cli/pull/674
  - abort update if cannot connect to internet by @Derisis13 in https://github.com/pystardust/ani-cli/pull/696
  - less curl in mp4upload by @CoolnsX in https://github.com/pystardust/ani-cli/pull/705
  - same ep fetching by @CoolnsX in https://github.com/pystardust/ani-cli/pull/708
  - fix:long access times by @CoolnsX in https://github.com/pystardust/ani-cli/pull/712

- Features
  - **ani-sync -s arg by @iamchokerman in https://github.com/pystardust/ani-cli/pull/694**
  - m3u8 quality with a different approach by @RaynardGerraldo in https://github.com/pystardust/ani-cli/pull/674
  - alternate links by @CoolnsX in https://github.com/pystardust/ani-cli/pull/704
  - ani-sync on windows by @iamchokerman in https://github.com/pystardust/ani-cli/pull/700

## New Contributors
* @WillPower3309 made their first contribution in https://github.com/pystardust/ani-cli/pull/707

**Full Changelog**: https://github.com/pystardust/ani-cli/compare/v2.1...v2.2

<hr /><em>This discussion was created from the release <a href='https://github.com/pystardust/ani-cli/releases/tag/v2.2'>v2.2</a>.</em>

**Comment by qubits9** [comment, marked as answer]
I liked this version, just one file, without libs.

**Reply by YusufKhan-gamedev** [reply]
Those are just 4 line functions, couldve easily been put in /bin

**Reply by justchokingaround** [reply]
https://github.com/pystardust/ani-cli/issues/773

**Reply by YusufKhan-gamedev** [reply]
I have no idea how the file system hierarchy being broken in ani-cli v3 is related here(its also still _kinda_ broken in the build instructions ;) ).

**Reply by justchokingaround** [reply]
internal modules/libraries shouldn't be in the binaries directory

**Reply by YusufKhan-gamedev** [reply]
They shouldnt be in the directory, they should all be in one file

**Reply by justchokingaround** [reply]
u can look at #727 for the whole lore, tho we are considering moving back to one file
